FiveCalls/FiveCalls/IssueContactDetail.swift

```diff
@@ -63,7 +63,7 @@ struct IssueContactDetail: View {
                         }
                     }
                 }.padding(.bottom)
-                Text(issue.markdownIssueScript)
+                Text(issue.markdownIssueScript(contact: currentContact, location: store.state.location))
```

Comment:
nit: can we migrate this to a computed property? Some benefits being better local testability (if we were ever to add unit tests here) and better adherence to observation if any of these other properties were to change. For example, if the location changed (however likely or unlikely this is), this text wouldn't be updated in its current form. In addition, here's an article explaining a good rule of thumb (use property for representing state or a value and use a method when invoking an action) when it comes to this: https://sarunw.com/posts/computed-property-vs-method/

Reply:
my nemesis, thanks for catching this style issue again